spec/classes/init_spec.rb

```diff
@@ -613,4 +613,34 @@
       })
     }
   end
+
+  context 'with parameter lookup_file_bind set' do
```

Comment:
Here is an outline for the test cases that would test the default inclusion of the class without setting lookup and with setting lookup to both valid and invalid entries on valid and invalid OS's. Could you please add these tests?

``` ruby
describe 'parameter lookup' do
  context 'set to default value' do
    context 'on OpenBSD' do
    end
    context 'on non-OpenBSD' do
    end
  end
  context 'set to a valid array' do
    context 'on OpenBSD' do
    end
    context 'on non-OpenBSD' do
    end
  end
  context 'set to an invalid type on a valid operatingsystem (OpenBSD)' do
  end
end
```